addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/serializer/CustomTankerkoenigDetailResultDeserializer.java

```diff
@@ -43,13 +43,15 @@ public TankerkoenigDetailResult deserialize(final JsonElement json, final Type t
             final Double e10 = jsonStation.get("e10").getAsDouble();
             final Double e5 = jsonStation.get("e5").getAsDouble();
             final Double diesel = jsonStation.get("diesel").getAsDouble();
+            final Boolean isOpen = jsonStation.get("isOpen").getAsBoolean();
```

Comment:
> I'm only using it in this line:
...
IMHO that way a null would be treated the same as "false".

No if I remember correctly, you call a setter which stores it to a boolean a few lines lower, that will cause an NPE.